### PR TITLE
Fix MoE related tests

### PR DESCRIPTION
**Pull request by RissyRan**
# Description

* Add weights normalization and update ckpt to unscanned version for faster decoding
* Disabled ckpt related tests for 8x22b until b/380148614 has been fixed (otherwise, with wrong ckpt, it always fails in XL ML)

If the change fixes a bug or a Github issue, please include a link, e.g.,:
FIXES: b/379315309

# Tests

* test locally
  ** 8x7b decoding: [link](https://paste.googleplex.com/6285026819047424)
  ** 8x7b forward_pass_logit_checker: [link](https://paste.googleplex.com/6038869945614336#l=468), [link](https://paste.googleplex.com/4652731477524480)

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run end-to-end tests tests and provided workload links above if applicable.
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: end_to_end/tpu/mixtral/8x7b/2_test_mixtral.sh [hunk 1]
```diff
@@ -24,20 +24,22 @@ export DATASET_PATH=gs://maxtext-dataset
...
+python3 MaxText/tests/forward_pass_logit_checker.py MaxText/configs/base.yml base_output_directory=${BASE_OUTPUT_PATH} load_parameters_path=${UNSCANNED_CKPT_PATH} run_name=matmul_forward_pass_test per_device_batch_size=8 model_name=mixtral-8x7b tokenizer_path=assets/tokenizer.mistral-v1 ici_tensor_parallelism=4 ici_fsdp_parallelism=16 max_prefill_predict_length=11 max_target_length=11 dataset_type=synthetic dtype=float32 megablox=False scan_layers=false --atol=3 --rtol=1 --token_size=4
 
 # Test whether the forward pass logits match the golden logits - megablox implementation
-python3 MaxText/tests/forward_pass_logit_checker.py MaxText/configs/base.yml base_output_directory=${BASE_OUTPUT_PATH} load_parameters_path=${SCANNED_CHECKPOINT} run_name=megablox_forward_pass_test per_device_batch_size=8 model_name=mixtral-8x7b tokenizer_path=assets/tokenizer.mistral-v1 ici_fsdp_parallelism=64 max_prefill_predict_length=11 max_target_length=11 dataset_type=synthetic dtype=bfloat16 weight_dtype=bfloat16 --atol=4 --rtol=1 --token_size=4
+# TODO(ranran): investigate the root cause of the excessive tolerance
```
Comment (gobbleturk) [inline end_to_end/tpu/mixtral/8x7b/2_test_mixtral.sh:38]: In b/380148614 the root cause of excessive tolerance?

**Review by gobbleturk — APPROVED** [review]

File: end_to_end/tpu/mixtral/8x7b/2_test_mixtral.sh (hunk 1 above)
Comment (RissyRan) [reply end_to_end/tpu/mixtral/8x7b/2_test_mixtral.sh:38]: I am not sure actually. Using current ckpt, matmul implementation could match the logits well. Something seems off, as we could match logits a while back.
